force-app/main/default/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.js

```diff
@@ -36,9 +36,11 @@ import insufficientPermissions from '@salesforce/label/c.commonInsufficientPermi
 import commonAdminPermissionErrorMessage from '@salesforce/label/c.commonAdminPermissionErrorMessage';
 
 import setGatewayId from '@salesforce/apex/PS_GatewayManagement.setGatewayId';
```

Comment:
A general comment on this JS class, how are we planning to integrate @screcco-sfdo changes here?

Reply:
As soon as his PR is merged, I will integrate the two.